__tests__/app.test.js

```diff
@@ -71,14 +70,53 @@ describe('appTests', () => {
                 
                 });
             })
-    
+        test('GET 200 /api/reviews accepts a category query, sort_by query, and order_query', () => {
+            return request(app)
+                .get('/api/reviews?category=social+deduction&sort_by=title&order=ASC')
+                .expect(200)
+                .then(({body})=>{
+                    const reviews = body.reviews;
+
+                    expect(Array.isArray(reviews)).toBe(true)
+                    expect(Object.prototype.toString.call(reviews[0])).toBe('[object Object]')
+
+                    expect(reviews).toBeSortedBy('title', {descending: false})
+
+                    reviews.forEach((review) => {
+                        expect(review.category).toBe('social deduction')
+                    })
+                })
+        })
+        test('GET 200: /api/reviews?category=social+deduction query defaults to sort by date and order descending if not specified', () => {
+            return request(app)
+                .get('/api/reviews?category=dexterity')
+                .expect(200)
+                .then(({body})=>{
+                    const reviews = body.reviews;
+
+                    expect(reviews).toBeSortedBy('created_at', {descending: true})
+                })
+        })
+        test('GET 200: /api/reviews?badlyformedquery Ignores a bad query, and returns default get reviews request.', () => {
+            return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then(({body})=>{
+                expect(Array.isArray(body.reviews)).toBe(true)
+                expect(Object.prototype.toString.call(body.reviews[0])).toBe('[object Object]')
+                expect(Object.keys(body.reviews[0]).length).toBe(9)
+            })
+        })
+        
+        
```

Comment:
Could we add another 200 test please for valid category query, but has no reviews responds with an empty array of articles?